snscrape/modules/instagram.py

```diff
@@ -19,6 +19,7 @@ class InstagramPost(snscrape.base.Item):
 	content: str
 	thumbnailUrl: str
 	displayUrl: str
+	mediaUrls: []
```

Comment:
This should be `typing.List[...]` with the type of the list elements inside (cf. below). Since PEP 585, `list` also works, but that requires Python 3.9, and I'd like to keep 3.8 support for the time being. A list object is always invalid though as it has to be a type.